alti/__init__.py

```diff
@@ -29,6 +30,9 @@ def main(global_config, **settings):
     config.add_route('checker', '/checker')
     config.add_route('checker_dev', '/checker_dev')
 
+    config.add_route('stats', '/stats')
```

Comment:
Maybe add examples to https://github.com/geoadmin/service-alti/blob/master/alti/static/index.html

Reply:
Or README.md